lib/index.js

```diff
@@ -212,10 +212,13 @@ Server.prototype = {
     }
     this.webApp = makeWebApp(webAppOptions);
     this.webServer.startListening(this.webApp.expressApp);
-    let pluginsLoaded = [];
+    let pluginsLoaded = 0;
+    let pluginCount = 0;
     this.pluginLoader.on('pluginAdded', util.asyncEventListener(event => {
+      pluginCount++;
       return this.pluginLoaded(event.data).then(() => {
         installLogger.info('Installed plugin: ' + event.data.identifier);
+        pluginsLoaded++;
```

Comment:
By looking at my logs, it appears this is a pretty late place in the plugin loading. If you were able to determine you loaded all plugins you wanted to load at this time, then you could print a final message.
Perhaps event.data could contain info that would tell you the % complete to be able to make that check.